pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md

```diff
@@ -33,7 +33,7 @@ The following table describes the list of platform applications that are deploye
 | cert-manager-1.7.1            | cert-manager          | True                |
 | external-dns-6.1.8            | external-dns          | False               |
 | fluent-bit-0.19.20            | fluent-bit            | False               |
-| gatekeeper-3.7.0              | gatekeeper            | False               |
+| gatekeeper-3.7.0              | gatekeeper            | True               |
```

Comment:
nit, misaligned rightmost column divider:
```suggestion
| gatekeeper-3.7.0              | gatekeeper            | True                |
```

Reply:
🔍 fixed in d0b7d4c33d2cbc28e6e04f3fb4e496b6b1c836a7!